integration/samples/core/specs/umd.ts

```diff
@@ -50,5 +50,12 @@ describe(`@sample/core`, () => {
       expect(API.AngularService).to.be.ok;
     });
 
+    it (`should downlevel iteration`, () => {
+      const iterable = function*() { yield* [1, 2, 3]; };
+      const values = API.AngularService.iterableToArray(iterable());
+
+      expect(values).to.deep.equal([1, 2, 3]);
```

Comment:
Hm, maybe I did not understand the downlevelIteration flag correct. The test verifies the `iterableToArray(input: Iterable<T>): Array<T>` method's public API. But shouldn't the test rather verify that the `__spread()` tslib helper is used internally (in the es5 bundle)? 

I guess the above test will always success even without downlevelIteration (since the `*.spec.ts` are run on node@6/7/8/9).

Reply:
I agree this test is up for debate, a textual contains of the source file may fit as well. But actually, the test will currently fail when the `downlevelIteration` flag is not used, I tested that :-)

The reason it fails is that without `downlevelIteration`, the method will have been compiled into `iterable.slice()` and that is an invalid call on the actual generator instance that is passed in the spec. I figured I might as well test the actual behaviour instead of verifying the source output.

Reply:
Ok, I trust you here!